framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2257,6 +2264,33 @@ public Void visitThrow(ThrowTree node, Void p) {
         return super.visitThrow(node, p);
     }
 
+    /**
+     * Rather than overriding this method, clients should often override {@link
+     * #visitAnnotatedType(List,Tree)}. That method also handles the case of annotations at the
+     * beginning of a variable or method declaration. javac parses all those annotations as being on
+     * the variable or method declaration, even though the ones that are type annotations logically
+     * belong to the variable type or method return type.
+     */
+    @Override
+    public Void visitAnnotatedType(AnnotatedTypeTree node, Void p) {
+        visitAnnotatedType(null, node);
+        return super.visitAnnotatedType(node, p);
+    }
+
+    /**
+     * Checks an annotated type. Invoked by {@link #visitAnnotatedType(AnnotatedTypeTree, Void)} and
```

Comment:
>`Checks an annotated type.`  

Is it `all` annotated types, or could we still be missing some?  It's fine if you aren't confident, I'm just curious.

Reply:
I'm pretty sure it is invoked for all annotated types.